### PR TITLE
[9.0] Web easy switch company v9

### PR DESCRIPTION
**Pull request by llacroix**

### CODE REVIEW

**Comment by llacroix** [comment]
https://github.com/OCA/web/pull/260 I guess this module should be used instead of mine.

**Comment by njeudy** [comment]
I think we have to merge request for this :)

**Comment by llacroix** [comment]
@njeudy I kind of improved the module now, it has an extra menu for odoo-enterprise (when navigating) and the company name is removed from the UserMenu.

**Comment by llacroix** [comment]
@njeudy  Apparently my bug with a menu missing has been solved. I think it has to do with the version of odoo-enterprise I tried. Now it works pretty much like your module.

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+web_easy_switch_company
+=======================
+
+This module extend web backend and allow user to switch to another company more easyly.
```
Comment (pedrobaeza) [inline README.rst:9]: s/extend/extends

Comment (pedrobaeza) [inline README.rst:9]: s/allow/allows

Comment (pedrobaeza) [inline README.rst:9]: s/easyly/easily

**Comment by hbrunn** [comment]
please rebase

**Comment by pedrobaeza** [comment]
Actually this one can be closed as the module has been already migrated.

**Pull request closed without merging**